### PR TITLE
Refactor polaris-labelled

### PR DESCRIPTION
**Pull request by vladucu**
### What's here

- refactors to ES6 class
- template-only components
- angle-bracket syntax

### CODE REVIEW

File: addon/components/polaris-labelled.js [hunk 1]
```diff
@@ -68,34 +51,67 @@ export default Component.extend({
...
     return error && typeof error !== 'boolean';
-  }).readOnly(),
-});
+  }
+
+  init() {
+    super.init(...arguments);
+
+    deprecate(
+      `[polaris-labelled] Passing 'dataTestLabelled' is deprecated! Switch to angle bracket invocation and pass an HTML attribute instead`,
```
Comment (andrewpye) [inline addon/components/polaris-labelled.js]: `:obama:` 😛

File: addon/templates/components/polaris-labelled.hbs [hunk 2]
```diff
@@ -1,37 +1,42 @@
...
-          onClick=(action action.onAction)
-        }}
+      {{#if @action}}
+        <PolarisButton
+          @plain={{true}}
+          @text={{@action.text}}
+          @disabled={{@action.disabled}}
+          @loading={{@action.loading}}
+          @accessibilityLabel={{@action.accessibilityLabel}}
+          @onClick={{action @action.onAction}}
```
Comment (andrewpye) [inline addon/templates/components/polaris-labelled.hbs]: HURRY UP AND GIVE US SPL@TTRIBUTES PLZ EMBER/GLIMMER 😅

File: tests/dummy/config/deprecation-workflow.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      handler: 'silence',
+      matchId: 'ember-polaris.polaris-description-list.description-component',
+    },
+    { handler: 'silence', matchId: 'computed-property.override' },
+    {
+      handler: 'silence',
+      matchId: 'ember-polaris.polaris-labelled.dataTestLabelled-arg',
+    },
+  ],
+};
```
Comment (andrewpye) [inline tests/dummy/config/deprecation-workflow.js]: Curious, why do we want to silence these in the dummy app? 🤔

Comment (vladucu) [reply tests/dummy/config/deprecation-workflow.js]: to reduce the noise while working through these PRs

**Comment by andrewpye** [comment]
You have some test failures that need to be taken care of before merging this 😉

**Comment by vladucu** [comment]
lmao, added a note about tests to most PRs but not this one

that happens with ember-beta....seems an issue unrelated to our code possibly related to https://github.com/ember-decorators/ember-decorators/pull/451

**Pull request closed without merging**